browserstack.yml

```diff
@@ -0,0 +1,71 @@
+# =============================
+# Set BrowserStack Credentials
+# =============================
+# Add your BrowserStack userName and acccessKey here or set BROWSERSTACK_USERNAME and
+# BROWSERSTACK_ACCESS_KEY as env variables
+userName: BROWSERSTACK_USERNAME
+accessKey: BROWSERSTACK_ACCESS_KEY
+
```

Comment:
source as a capability for attribution is missing?

Reply:
Hardcoded inside capabilties rather than in yml file